virttest/guest_agent.py

```diff
@@ -885,6 +885,43 @@ def get_disks(self):
         self.check_has_command(cmd)
         return self.cmd(cmd)
 
+    def ssh_add_authorized_keys(self, username, *keys, **kwargs):
+        """
+        Add ssh public keys into guest-agent.
+
+        :param username: username of host
+        :param keys: value of public keys
+        :param kwargs: optional keyword arguments
+        :return: command handle
+        """
+        cmd = "guest-ssh-add-authorized-keys"
+        keys = list(keys)
+        reset = kwargs.get('reset')
```

Comment:
Does it need a default value? And, seems no other arguments besides `reset`, not necessary to have `**kwargs`.

Reply:
for this part of code, It always got CI failed if we still use original method "def ssh_add_authorized_keys(self, username, *keys, reset=None",  the CI error output was warning that there are syntax error behind *key. it might our CI code checking syntax. so I wrote code in this way.